bindings/c/CMakeLists.txt

```diff
@@ -26,6 +26,90 @@ endif()
 
 option(TEST_ENABLE_ASAN "Enable AddressSanitizer for tests" OFF)
 
+
+option(SERVICES_AZBLOB "Enable Azblob service" OFF)
+option(SERVICES_AZDLS "Enable Azdls service" OFF)
+option(SERVICES_COS "Enable Cos service" OFF)
+option(SERVICES_FS "Enable Fs service" OFF)
+option(SERVICES_GCS "Enable Gcs service" OFF)
+option(SERVICES_GHAC "Enable Ghac service" OFF)
+option(SERVICES_HTTP "Enable Http service" OFF)
+option(SERVICES_IPMFS "Enable Ipmfs service" OFF)
+option(SERVICES_MEMORY "Enable Memory service" OFF)
+option(SERVICES_OBS "Enable Obs service" OFF)
+option(SERVICES_OSS "Enable Oss service" OFF)
+option(SERVICES_S3 "Enable S3 service" OFF)
+option(SERVICES_WEBDAV "Enable Webdav service" OFF)
+option(SERVICES_WEBHDFS "Enable Webhdfs service" OFF)
+option(SERVICES_AZFILE "Enable Azfile service" OFF)
+
+
+# Build the list of Rust features based on enabled services
+set(RUST_FEATURES "") # Initialize empty list
+
+if(SERVICES_ALL)
```

Comment:
It's impractical for bindings to maintain such a list. Can we simply allow users to pass `services-s3,services-fs` directly?